source/templates.tex

```diff
@@ -1280,7 +1280,7 @@
 
 struct A {
   constexpr A(const char*) {}
-  auto operator<=>(A, A) = default;
+  friend auto operator<=>(A, A) = default;
```

Comment:
Can you add a `const&` to both parameters here too? Thanks

Reply:
Good point.

Reply:
Done, thanks. I had been unaware that the pass-by-value form was "valid, but not `=default`-able." Tim pointed me at [[class.compare.default]](http://eel.is/c++draft/class.compare.default) and set me straight.

I guess I'm still dubious of

    struct C;
    std::strong_ordering operator<=>(const C&, const C&);
    struct C {
        friend std::strong_ordering operator<=>(const C&, const C&);
    };
    std::strong_ordering operator<=>(const C&, const C&) = default;

IIUC, this `C` has a defaulted `operator<=>` but it is not structural (because it was not defined as defaulted *in the definition of `C`*) and incidentally is not `constexpr` either. I would feel better if this `C` were ill-formed. (Like, if `=default` were permitted only inside the definition of `C`, for these new operators.)